EIPS/eip-5375.md

```diff
@@ -0,0 +1,293 @@
+There is currently no unified protocol to identify the author of an NFT. Some existing techniques include:
+- Using the mint transaction signer
+  - Requires that the minter and the author are the same
+  - Does not support multiple authors
+- Using the first Transfer event for a given ID
+  - Contract/minter can claim that someone else is the author without their consent
+  - Does not support multiple authors
+- Using a custom method/custom JSON field
+  - Requires per-contract support by NFT platforms
+  - Contract/minter can claim that someone else is the author without their consent
+
+The first practice is the most common, mainly due to the fact that relying on minters to provide truthful information opens up avenues for frauds (e.g. selling an NFT while claiming that it was made by a famous artist). However, there are several situations where the minter and the author might not be the same, such as:
+- NFTs minted by a contract
+- Lazy minting
+- NFTs minted by an intermediary (which can be particularly useful when the author is not tech-savvy and/or the minting process is convoluted)
+
+This document thus defines a standard which allows the minter to provide authorship information, while also preventing authorship claims without the author's consent.
+
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### Definitions
+
+- **Authors**: creators of an NFT
+- **Minter**: entity responsible for the actual minting transaction; the minter and the authors MAY be the same
+- **Verifier**: entity that wants to verify the authorship of an NFT (e.g. a user or an NFT marketplace)
+- **Author Consent Proof (ACP)**: a signed message that proves that the signer agrees to be considered the author of the NFT
+
+### Authorship Support
+
+The standard introduces a new JSON field, named `authorInfo`. It provides a REQUIRED interface for authorship claiming, as well as an OPTIONAL interface for author consent proofs.
+
+`authorInfo` is a top-level field of the NFT metadata. Specifically:
+- If a contract supports the metadata extension for EIP-721, the JSON document pointed by `tokenURI(uint256 _tokenId)` MUST include the top-level field `authorInfo`
+- If a contract supports the metadata extension for EIP-1155, the JSON document pointed by `uri(uint256 _id)` MUST include a top-level field `authorInfo`
+
+The JSON schema of `authorInfo` (named `ERC5375AuthorInfoSchema`) is defined as follows:
+```json
+{
+    "type": "object",
+    "properties": {
+        "consentInfo": {
+            "type": "object",
+            "description": "Helper fields for consent verification",
+            "properties": {
+                "chainId": {
+                    "type": "",
+                    "description": "EIP-155 chain id"
+                },
+                "id": {
+                    "type": "",
```

Comment:
Same here, shouldn't this be a type?

Reply:
Fixed in 8a512e6194c1b84a142b6aaf0ea7cee0a83705f8.